### PR TITLE
add new logging lines

### PR DESCRIPTION
**Pull request by jeffnvidia**
## Summary
Based on PR 21
add some lines to better see the potential errors when running cloudAI

## Test Plan
Test by @amaslenn
CI
Test by @jeffnvidia
2.1 Slurm command generation
$ cloudaix --mode run --system_config_path conf/v0.6/general/system/israel_1.toml --test_scenario_path conf/v0.6/general/test_scenario/llama/llama.toml

### CODE REVIEW

File: tests/test_slurm_system.py
Comment (amaslenn) [inline tests/test_slurm_system.py:1]: Those are unrelated changes.

Comment (jeffnvidia) [reply tests/test_slurm_system.py:1]: I'm just rebasing on another PR it's not to be taken independently

**Comment by TaekyungHeo** [comment]
I believe https://github.com/NVIDIA/cloudai/pull/41 covers this PR. You can close this PR.

cc: @srinivas212, @amaslenn
